corehq/apps/export/utils.py

```diff
@@ -0,0 +1,8 @@
+from .const import TRANSFORM_FUNCTIONS
+from .exceptions import ExportInvalidTransform
+
+
+def is_valid_transform(value):
+    for transform in value:
+        if transform not in TRANSFORM_FUNCTIONS:
+            raise ExportInvalidTransform('{} is not a valid transform'.format(value))
```

Comment:
Oh good point, I didn't think of that. Yeah I'm not sure if it is smart enough...